content/03.use-cases-a.md

```diff
@@ -1,45 +1,40 @@
-## 12 Practical ways of using GitHub in EEB
+## Twelve practical ways GitHub can accelerate research in ecology and evolution
 
-### Storing a research compendium
+### Storing and sharing research compendia
 
-<!--*contributors to this section: Dylan Gomes, Emma Hudgins* -->
-In EEB, a research compendium includes all computational materials related to research production, including data, code for analyses and protocols.
-Having multiple copies of these files is important to ensure that research can continue should one copy of a file be accidentally modified or deleted.
-Many researchers begin using GitHub as a means to backup their research compendium [@doi:10.1080/00031305.2017.1375986] to a remote server (pull-commit-push, see [Box 1](#definitions)).
-The GitHub repository serves as a centralized backup and also allows users to synchronize files and work on any device with internet access. 
-Because GitHub is built on git, version control features are also available, allowing users to re-visit or even restore previous versions of the repository.
-Storing a research compendium as a GitHub repository has additional benefits such as facilitating collaboration, integration with data and code archiving services, and contribution to open science, all discussed in sections below.
+<!--*contributors to this section: Dylan Gomes, Emma Hudgins, Pedro Braga* -->
+An EEB research compendium includes all computational materials related to research production, including data, code for analyses and protocols.
```

Comment:
Added EEB back in. We want to start sections tying in what we are saying back into the utility specific within EEB as much as possible